docker/README.md

```diff
@@ -0,0 +1,37 @@
+# Running Oasis node and gateway in Docker
+
+This directory contains a Dockerfile that builds an image containing both
+the [Oasis node][0] and the [Oasis Rosetta gateway][1], as instructed by the
+[Rosetta requirements][2].
+
+The node should be configured by following the instructions at:
+https://docs.oasis.dev/operators/joining-the-testnet.html
+
+The `/serverdir` in the instructions is equivalent to the `/data` mountpoint
+of the Docker image.
+
+Certain steps are not needed, e.g. provisioning entities and nodes.
+
+You can use the `config.yml` file included in this directory and get the
+`genesis.json` file from the instructions above.
+
+Don't forget to set the proper permissions on the directory you're using as
+the `/data` mountpoint, as well as its subdirectories and files!
+
+To build the Docker image:
+
+	docker build .
```

Comment:
Could suggest passing `-t oasis-core-rosetta-gateway` so you don't need to use `IMAGE_HASH_FROM_BUILD_STEP` below.

Reply:
Good idea, fixed!